lib/cucumber/core/gherkin/document.rb

```diff
@@ -3,11 +3,12 @@ module Cucumber
   module Core
     module Gherkin
       class Document
-        attr_reader :uri, :body
+        attr_reader :uri, :body, :language
 
-        def initialize(uri, body)
-          @uri = uri
-          @body = body
+        def initialize(uri, body, language=nil)
```

Comment:
It is a question about where the default language should be defined. One option is to let gherkin do that, that is only when the "-L" option is used cucumber-ruby pasess a language to the parser, otherwise gherkin will use its default (that is "en"). Another option is that cucuber-ruby defines the default (a default that can differ from gherkins default), which means that cucumber-ruby even when without the "-L" option passes a language to the parser (which should be "en").